core/src/main/kotlin/io/specmatic/core/pattern/StringPattern.kt

```diff
@@ -67,36 +67,27 @@ data class StringPattern (
         return JSONArrayValue(valueList)
     }
 
-    private val randomStringLength: Int =
+    private val patternMinLength: Int =
         when {
-            minLength != null && 5 < minLength -> minLength
-            maxLength != null && 5 > maxLength -> maxLength
+            minLength != null && minLength > 0 -> minLength
+            maxLength != null && maxLength < 5 -> 1
```

Comment:
This worked a little differently earlier. If maxLength < 4, use it, else use upto 5 chars. What's the reason we changed this?

Reply:
If minimum and maximum is both supplied, pattern length would be in range of (min to max)
If minimum length is not supplied in spec -> and maximum is less than default (5), pattern would be of length (1 i.e. generex min value, max)
Say max = 4 is only supplied
pattern length would be in range (1-4)

Reply:
We revamped this logic a bit as per the one given above